21_pattern_examples.md

```
@@ -256,6 +256,152 @@ int main() {
+            else
+                printf("  ");
+
+        }
+        printf("\n");
+    }
+    return 0;
+}
+```
+
+**Expected output:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)